ingestion/tests/cli_e2e/README.md

```diff
@@ -0,0 +1,95 @@
+# E2E CLI tests
+
+Currently, it runs CLI tests for any database connector. 
+
+- `test_cli_db_base` has 8 test definitions for database connectors. It is an abstract class.  
+- `test_cli_db_base_common` is another abstract class which for those connectors whose sources implement the `CommonDbSourceService` class. 
+- It partially implements some methods from `test_cli_db_base`.
+- `test_cli_{connector}` is the specific connector test. More tests apart the ones implemented by the `test_cli_db_base` can be run inside this class.
+
+## How to add a database connector
+
+1. Use `test_cli_mysql.py` as example. Your connector E2E CLI test must follow the name convention: `test_cli_{connector}.py` and the test 
+class must extend from `CliCommonDB.TestSuite` if the connector's source implement the `CommonDbSourceService` class, otherwise, from `CliDBBase.TestSuite`.
+
+2. Add an ingestion YAML file with the service and the credentials of it. Use when possible a Dockerized environment, otherwise, remember to use environment
+variables for sensitive information in case of external resources.
```

Comment:
Maybe we can add a line about how the different YAMLs get generated automatically for each workflow run